src/gg.regression.unity.bots/Runtime/Scripts/RGSettings.cs

```diff
@@ -18,8 +19,9 @@ public class RGSettings : ScriptableObject
     {
         // We store this asset in the Resources folder so that it can be accessed from any build
         // Note that it can only be created in the editor, however.
+        private static readonly string SETTINGS_DIRECTORY = "Assets/Resources";
         private static readonly string SETTINGS_RESOURCE_NAME = "RGSettings";
-        private static readonly string SETTINGS_PATH = $"Assets/Resources/{SETTINGS_RESOURCE_NAME}.asset";
+        private static readonly string SETTINGS_PATH = $"{SETTINGS_DIRECTORY}/{SETTINGS_RESOURCE_NAME}.asset";
```

Comment:
do we need to construct this path string in a platform-generic way? `/` vs `\`

Reply:
luckily no, the api accepts / prefers `/`